src/net.cpp

```diff
@@ -1260,6 +1260,7 @@ void CConnman::ThreadSocketHandler()
     unsigned int nPrevNodeCount = 0;
     while (!interruptNet)
     {
+        InactivityChecks();
```

Comment:
In commit "Run InactivityChecks before other socket handling logic." (8491dfbadceb8052f852ebf0020a622194db525f)

It seems like this commit should have no effect, but if it has no effect, then it's not clear why it would be desirable? I think it would be helpful if the commit messages in this PR said explicitly whether the commit is or isn't intending to change behavior.

Reply:
It doesn't really matter, but running it earlier might allow for marking nodes disconnected before the socket handling logic runs.